### DISCUSSION TITLE
第5节课笔记(8班)

### DISCUSSION THREAD

**Discussion by Hongru0306** [InternLM/Tutorial#312] [笔记]
请在下方提交笔记链接,格式为
-->报名序号+连接:
例如:213+https://kvudif1helh.feishu.cn/docx/...

**Comment by zhaoxin111** [comment]
445+https://app.yinxiang.com/fx/9ac56e91-c73c-4b09-8e97-26e67d011205

**Comment by baijiesong** [comment]
509+https://github.com/baijiesong/InternLM_Learning/blob/main/%E7%AC%AC%E4%BA%94%E8%8A%82%E7%AC%94%E8%AE%B0.md

**Comment by Yujin320** [comment]
587+https://github.com/Yujin320/InternLM-hyx-Learning/blob/main/handwrite05.md

**Comment by NYYY0421** [comment]
501+https://github.com/NYYY0421/homework_lmdeploy/blob/main/note5.md

**Comment by superkong001** [comment]
545+https://github.com/superkong001/InternLM_Learning/blob/main/howework4_LMDeploy_note.md

**Comment by jamiechoi1995** [comment]
554+https://blog.csdn.net/sinat_26253653/article/details/135622009

**Comment by Cyydz** [comment]
527+https://www.wolai.com/3oPwED5Ky7L7a9Cfa2vCA6

**Comment by hzsun1995** [comment]
446+https://github.com/hzsun1995/internlm-course/blob/main/course5_note.md

**Comment by BJUT-WangZhengTao** [comment]
371+https://blog.csdn.net/m0_54169072/article/details/135599186

**Comment by Alias-z** [comment]
388+http://t.csdnimg.cn/iTpTc

**Comment by JieGenius** [comment]
555+https://github.com/JieGenius/InternLMLearning/blob/main/lesson5/note.md

**Comment by carlinl** [comment]
596 + https://pt35a5ibej.feishu.cn/docx/WSBQdFspwoRLqRxilAXcFpf6nHg?from=from_copylink

**Comment by 00z768** [comment]
1095 + https://github.com/00z768/model/blob/main/LMDeploy.md

**Comment by ZPfree** [comment]
605+https://github.com/ZPfree/homework/blob/main/5note.md

**Comment by sunxiaobei** [comment]
585+https://github.com/sunxiaobei/InternLM_tutorial/blob/main/notes/Day05.md

**Comment by L-369-maha** [comment]
521+https://github.com/L-369-maha/InternLM/blob/main/Five-lmdeploy/Note.md

**Comment by Doublefei-Giser** [comment]
409+https://blog.51cto.com/u_16507376/9392548
